packages/build-tools/src/steps/functions/repack.ts

```diff
@@ -163,10 +163,19 @@ function createSpawnAsyncStepAdapter({
     args: string[],
     options?: SpawnProcessOptions
   ): SpawnProcessPromise<SpawnProcessResult> {
-    return spawnAsync(command, args, {
+    const promise = spawnAsync(command, args, {
       ...options,
       ...(verbose ? { logger, stdio: 'pipe' } : { logger: undefined }),
```

Comment:
Would it make sense to forward `{ logger, stdio: 'pipe' }` always, no matter `verbose`? What does `repack` print out? Maybe it's interesting?

Reply:
the verbose log is just too much. created an example for you to check https://expo.dev/accounts/kudochien/projects/sdk53/workflows/019816f7-57c4-7321-84ab-ecbf0f6f7a66